website/source/docs/providers/aws/r/network_acl.html.markdown

```diff
@@ -33,6 +33,10 @@ resource "aws_network_acl" "main" {
 		from_port = 80
 		to_port = 80
 	}
+
+	tags = {
```

Comment:
Actually I thought so, but in this document `ingress` `egress` explanation use `=` too, I followed those.

I think it is better to integrate code format, could I remove `=` from `ingress` and `egress` at this PR?